content/trial_getting_started/set_trial_manager/other-deployments.md

```diff
@@ -7,36 +7,76 @@ alwaysopen = false
 
 {{%children style="h2" description="true"%}}
 
-This page provides download links to all of Cloudify's deployment packages, both Premium and Community.
+This page provides download links to all of {{< param product_name >}}'s deployment packages, both Premium and Community.
 
-Read about the differences between our [Cloudify versions](https://cloudify.co/download).
+Read about the differences between our {{< param product_name >}} versions, and get an **activation key** at [https://cloudify.co/download]](https://cloudify.co/download).
 
 
 
 
-## Cloudify Premium (Premium download / Hosted service trial)
+## {{< param product_name >}} Premium (Premium download / Hosted service trial)
 
-Current Version: 5.1       [Release Notes](https://cloudify.co/release-notes-5-1-2020/)
+### 5.1.1
 
-#### Downloads
+v5.1.1 is the latest __minor__ release. It can be used for new installations, upgrade from 5.0.5 or earlier releases, or as an update to a v5.1.0 system.
```

Comment:
Indeed 5.1.2, 5.1.3 are patch versions.
however, 5.1.1 is a minor release. It was badly versioned when created because the numbering concept was updated after 5.1.1 was already created.